data/korath/korath outfits.txt

```diff
@@ -178,6 +205,294 @@ outfit "Control Transceiver"
 	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."
 
 
+outfit "Afterburner (Asteroid Class)"
+	plural "Afterburners (Asteroid Class)"
+	category "Engines"
+	"cost" 136000
+	thumbnail "outfit/tiny korath afterburner"
+	"mass" 9
+	"outfit space" -9
+	"engine capacity" -5
+	"fuel capacity" 100
+	"ramscoop" 0.25
+	"afterburner thrust" 19.2
+	"afterburner energy" 0.616
+	"afterburner heat" 6.26
+	"afterburner fuel" 0.03
+	"afterburner effect" "korath afterburner tiny"
+	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+	description "The smallest of afterburners is as much fuel processor as it is afterburner. It allows Korath small craft operators to squeeze a hyperdrive into otherwise intrastellar ships and dispense with conventional thrusters entirely."
+
+outfit "Afterburner (Comet Class)"
+	plural "Afterburner (Comet Class)"
+	category "Engines"
+	"cost" 299000
+	thumbnail "outfit/small korath afterburner"
+	"mass" 19
+	"outfit space" -19
+	"engine capacity" -19
+	"afterburner thrust" 78
+	"afterburner energy" 2.24
+	"afterburner heat" 22.75
+	"afterburner fuel" 0.1
+	"afterburner effect" "korath afterburner small"
+	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+	description "Adding these afterburners to an interceptor allows it to reach absurd speeds; however, the heat produced ensures that running them for any substantial period of time will require some serious cooling."
+
+outfit "Afterburner (Lunar Class)"
+	plural "Afterburners (Lunar Class)"
+	category "Engines"
+	"cost" 647000
+	thumbnail "outfit/medium korath afterburner"
+	"mass" 32
+	"outfit space" -32
+	"engine capacity" -32
+	"afterburner thrust" 148.8
+	"afterburner energy" 3.92
+	"afterburner heat" 41.65
+	"afterburner fuel" 0.18
+	"afterburner effect" "korath afterburner medium"
+	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, you can't help but feel concerned about the safety of installing one."
```

Comment:
```suggestion
	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, the prospect of installing one brings to mind many safety concerns."
```
It's best to avoid pronouns in descriptions, I think. This feels a tiny bit better to me, at least.